src/log.js

```diff
@@ -30,17 +30,17 @@ const start = Date.now();
  *
  * @const {string}
  */
-export const USER_ERROR_SENTINEL = '\u200B\u200B\u200B';
-
 
 /**
- * @return {boolean} Whether this message was a user error.
+ * @return {boolean} Whether this error was a user error.
  */
-export function isUserErrorMessage(message) {
-  return message.indexOf(USER_ERROR_SENTINEL) >= 0;
+export function isUserError(error) {
+  if (error == undefined) {
```

Comment:
@zhouyx in test-error, sometimes they call getErrorReportUrl with error undefined and this function also calls isUserError()